src/components/TextField/TextField.tsx

```diff
@@ -44,7 +44,7 @@ export const TextField: React.FC<Props> = ({
             'is-danger': hasError,
           })}
           placeholder={placeholder}
-          value={value}
+          value={value.trim()}
```

Comment:
try to type more than one word in title or description. it is impossible now .
the reason is here
```suggestion
          value={value}
```